### DISCUSSION TITLE
The calendar widget needs a "jump to date" feature like the Nokia 7110.

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#664] [Ideas]
The Nokia 7110, an 1999 mobile phone, has a calendar with features a "jump to date" feature.

Samsung had this feature in its "S Planner" app but removed it with the S6 (which lacked what feels like half the features of the S5 anyway).

Please add a "jump to date" feature to the calendar widget. If an 1999 mobile phone has it, a 2024 calendar widget should too.

**Comment by anaximeno** [comment]
You should give more info on how it would work, etc.
